docs/docs/aws.md

```diff
@@ -669,6 +665,63 @@ Users can use catalog properties to override the defaults. For example, to confi
 --conf spark.sql.catalog.my_catalog.http-client.apache.max-connections=5
 ```
 
+## Using the Hadoop S3A Connector
+
+The Apache Hadoop S3A Connector is an alternaive to S3FileIO. It:
+* Uses an AWS v2 client qualified across a broad set of applications, including Apache Spark, Apache Hive, Apache HBase and more. This may lag the Iceberg artifacts.
+* Contains detection and recovery for S3 failures beyond that in the AWS SDK -recovery added "one support call at a time".
+* Supports scatter/gather IO "Vector IO" for high-performance Parquet reads.
```

Comment:
Is vectored IO supported in the Iceberg read path?  I wouldn't callout anything that isn't supported natively.